src/refereecore.cpp

```diff
@@ -0,0 +1,25 @@
+#include "refereecore.h"
+
+RefereeCore::RefereeCore() {
+    // Creating world pointer
+    _world = new World();
+}
+
+RefereeCore::~RefereeCore() {
+    // Deleting world module
+    delete _world;
+}
+
+void RefereeCore::start() {
+    // Creating vision pointer and adding it to world with priority 0
```

Comment:
The **start()** method in **RefereeCore** class will create the module pointers and insert them into World module